BazelExtensions/xcode_configuration_provider.bzl

```diff
@@ -84,10 +84,14 @@ def _extract_generated_sources(target, ctx):
     if ctx.rule.kind == "entitlements_writer":
         files.append(target.files)
 
+    include_swift_outputs = ctx.attr.include_swift_outputs == "true" 
```

Comment:
This is only used within the next conditional so move it inside that block?

Reply:
Good point, will update 👍